projects/netgrif-components-core/src/lib/allowed-nets/services/base-allowed-nets.service.ts

```diff
@@ -30,6 +30,7 @@ export class BaseAllowedNetsService implements OnDestroy {
     }
 
     /**
+     * @deprecated
```

Comment:
the deprecated annotation should indicate the replacement

Reply:
Added suggestion to deprecated annotation.